blog/_posts/2020-1-23-crash-course-kubernetes-nodes.md

```diff
@@ -0,0 +1,168 @@
+		<div class="slide" data-image="/assets/images/crash-course-kubernetes-nodes/slide5.png"></div>
+		<div class="slide" data-image="/assets/images/crash-course-kubernetes-nodes/slide6.png"></div>
+		<div class="slide" data-image="/assets/images/crash-course-kubernetes-nodes/slide7.png"></div>
+		<div class="slide" data-image="/assets/images/crash-course-kubernetes-nodes/slide8.png"></div>
+		<div class="slide" data-image="/assets/images/crash-course-kubernetes-nodes/slide9.png"></div>
+		<div class="slide" data-image="/assets/images/crash-course-kubernetes-nodes/slide10.png"></div>
+	</div>
+	<div class="slider-nav"><!-- "Previous" and "Next" actions -->
+		<button class="slider-previous">Previous</button>
+		<button class="slider-next">Next</button>
+	</div>
+</div>	
+
+
+To read more about the above components, and all the other bits and pieces on the master node
+that do important work, see [the Kubernetes docs](https://kubernetes.io/docs/concepts/overview/components/#master-components).
+
+### Worker Node(s)
+All the fine-grained control of the master node does no good if there’s nowhere for your apps to
+run. That’s where worker nodes come in. Worker nodes are machines that run one or more pods
+(i.e. units of your app).
+
+If you’re connected to a Kubernetes cluster in the cloud, you’ll most likely run your pods across
+multiple nodes. If you’re running Kubernetes locally (e.g. Docker Desktop, KinD, Minikube), you’ll
+be working with a single node, which is a VM running on your computer. Regardless of the number
+of worker nodes you have at your disposal, and whether they’re physical or virtual machines, the
+basic anatomy is the same.
+
+In addition to your pod(s), a worker node has some additional daemons that allow it to do its job:
+
+* **Kubelet**: makes sure that the right containers are running on the right pods. It works a lot
+like the controllers that live on the master node: the kubelet reads the state of all the pods
+on its node, compares that against the desired state and, if there’s a discrepancy, makes the necessary changes
+* **Kube-proxy**: proxies network traffic to/from pods (since pods aren’t themselves directly
+connected to the network)
+
+[Read more about worker nodes and their components here](https://kubernetes.io/docs/concepts/overview/components/#node-components).
+
+A worker node is only as good as the pods it’s running, so how does it figure out which pods to
+run? This is where the pod scheduler, our last piece of master node machinery, comes in.
+
+#### Scheduling: Putting Pods on Nodes
+The **pod scheduler** on the master node watches for new pods that haven’t been assigned to
+nodes. These are pods that need creation, but we don’t yet know what computer they’re going to
+live on. For each of these pods, the pod scheduler finds the “best” node to put it on.
+
+First, it checks for _possible_ nodes, that is, those that have enough resources. With the list
```

Comment:
Maybe "Then, with the list..."? It feels slightly awkward to have a "First, " and then nothing indicating we've moved onto "second"

I dunno, this whole paragraph feels weird:
* no motivation of why one should care about this
* it kinda felt like I was waiting for the "second," after the "first,", though that's obviously fixable
* the whole paragraph just kinda ends, without really connecting it to anything, which is then extra weird because it's basically the end of the blog post

Maybe that's kind of crappy feedback because I don't have a concrete suggestion, but I'm not sure if it'd be better to cut this paragraph or find a way to make it relevant, and don't have good ideas on how to make it relevant.

Reply:
good call, i moved this up and will add a better closer to this post